instrumentation/sequel/lib/sequel/extensions/opentelemetry.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'sequel'
+
+# Sequel: The Database Toolkit for Ruby
+module Sequel
+  # The Sequel::Opentelemetry extension
+  #
+  # @example Simple usage
+  #
+  #     require 'sequel-opentelemetry'
+  #     DB.extension :opentelemetry
+  #
+  # Related module: Sequel::Opentelemetry
+  module Opentelemetry
+    def tracer
+      @tracer ||= ::OpenTelemetry.tracer_provider.tracer('Sequel', ::Sequel::VERSION)
+    end
+
+    def trace_execute(super_method, sequel_method, sql, options, &block)
+      response = nil
+      attributes = {
+        'component' => 'Sequel',
+        ::OpenTelemetry::SemanticConventions::Trace::DB_NAME => opts[:database],
+        ::OpenTelemetry::SemanticConventions::Trace::DB_SYSTEM => database_type.to_s,
+        ::OpenTelemetry::SemanticConventions::Trace::DB_STATEMENT => sql,
```

Comment:
🚨 Recording the Raw SQL statement will leak PII and other kinds of sensitive data. 

If the `sequel` gem does not support a way to provide a templated SQL statement then we must omit it. 

@kaylareopelle has been working on a PR to extract reusable SQL obfuscation code here: https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/529

Once that is merged, then you may update this instrumentation to optionally use it.

Reply:
I suspect the template is available. Will find it.

Reply:
No template available. Removed the sql.

Reply:
Or we can look at @kaylareopelle 's SQL Obfuscation gem?